test/integration/completed_transaction_test.rb

```diff
@@ -2,11 +2,32 @@
 require 'integration_test_helper'
 
 class CompletedTransactionTest < ActionDispatch::IntegrationTest
+  setup do
+    @payload = {
```

Comment:
Great, could do a similar clean up in 'test/functional/completed_transaction_controller_test.rb' to be consistent?